include/image-commands.mk

```diff
@@ -355,6 +355,21 @@ define Build/tplink-v2-image
 	rm -rf $@.new
 endef
 
+# Overwrite the first 160 bytes of the image with wrgg header.
+# Make room for this header by building the image with e.g. "pad-extra 160 | append-kernel | ..."
+define Build/wrgg-image
+	dd if=$@ of=$@.new iflag=skip_bytes skip=160
```

Comment:
This looks like it removes the 160 that got added by pad-extra?! Do you have to do this because you need to have the correct padding for the "pad-to 8M" / "pad-to 64k" down in your image generation rules? If so, then maybe "pad-offset" (see how the the DIR-685 or DAP-2695-A1 does it) is the way to go.

Reply:
I originally felt this was a less fragile way than correctly handling the offset in every following step of the image rule. But sure, this works just as well, and with the benefit of being the same as the other firmwares.